### ISSUE TITLE
Fast Group Muter/Bypass - Not appearing on API json file

### ISSUE THREAD

**Issue by SilverRaven03** [rgthree/rgthree-comfy#124]
Would be great if the Fast Group Muter/Bypass would appear in the .json saved workflow for API, I saw that several nodes in the past didn't appear there and now some are doing so but not this particular node.

**Comment by rgthree** [comment]
Fast Muter, etc. are what's called a "virtual node", which is a node that is only in the UI and the backend doesn't need in order to evaluate the prompt, thus, when ComfyUI sends the graph to the backend via the API, it skips this (and other) virtual nodes.

When you "save as API" it uses this same mechanism; the json is what the backend recieves.

I'm curious what the reason to be fore this nodes to be sent to the backend?

**Comment by SilverRaven03** [comment]
You're completely right, I understood later that anything bypassed in the workflow will just not appear in the API .json, just trying to find an appropriate way to be able to run a workflow via API where the same workflow can be used to run two different paths chosen by a integer. This all without give an error as some custom nodes seem to give an error when the execution path is cut by a simple "If" node or "Switch" node.

**Comment by julien-blanchon** [comment]
@SilverRaven03 Did you found a solution ?

**Comment by kudou-reira** [comment]
> @SilverRaven03 Did you found a solution ?

I'm also interested in this.

**Comment by Expertaz** [comment]
@julien-blanchon @kudou-reira @SilverRaven03 @rgthree Hey, did you found a solution ?

**Comment by niyuanfa** [comment]
@rgthree Because I want comfyui to dynamically modify the json data automatically by changing the value of the node on the back end

**Comment by drmbt** [comment]
still curious about this, i know there is talks of a pull request to reverse the flow making this supported, but is there currently any way to an export an API json that would allow me to do things like conditionally choose to bypass the time associatedwith loading models, i.e. controlnet, lora, ipadapter, upscaler? 

I want a workflow that has all of these things in it already, but can be lean and fast if i don't need them

**Comment by stephennilor** [comment]
> still curious about this, i know there is talks of a pull request to reverse the flow making this supported, but is there currently any way to an export an API json that would allow me to do things like conditionally choose to bypass the time associatedwith loading models, i.e. controlnet, lora, ipadapter, upscaler?
> 
> I want a workflow that has all of these things in it already, but can be lean and fast if i don't need them

I've been using very cumbersome Switch Any based workflows (from Crystools) but this would be far superior, even if it just means being able to convert the Fast Muter widgets to inputs and manipulate it from elsewhere
